react/__mocks__/vtex.pixel-manager/PixelContext.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export function Pixel(Comp) {
```

Comment:
Why did u add just the mock and not the test?

Reply:
I added the mock because some tests were breaking after my changes.